src/ergodex-cdk/styles/variables.less

```diff
@@ -168,6 +168,18 @@
                              0 6px 16px rgba(0, 0, 0, 0.08),
                              0 3px 6px -4px rgba(0, 0, 0, 0.12);
 
+  // Token Select
+  --ergo-tokenselect-default-bg: @light-color-orange-6;
+  --ergo-tokenselect-lbl-color: @light-color-gray-1;
+
+  --ergo-tokenselect-hover-bg: @light-color-orange-5;
+  --ergo-tokenselect-active-bg: @light-color-orange-7;
+
+  --ergo-tokenselect-selected-background: @light-color-gray-1;
+  --ergo-tokenselect-selected-color: @light-color-gray-9;
+  --ergo-tokenselect-selected-hover-bg: @light-color-gray-3;
+  --ergo-tokenselect-selected-active-bg: @light-color-gray-2;
+
```

Comment:
The light version of TokenSelect component has a box shadow.
It is the same as `--ergo-message-box-shadow`
You can rename it to avoid duplication.